src/audio/Kconfig

```diff
@@ -75,4 +75,118 @@ config COMP_TEST_KEYPHRASE
 	  Select for KEYPHRASE_TEST component.
 	  Provides basic functionality for use in testing of keyphrase detection pipelines.
 
-endmenu
+config COMP_ASRC
+	bool "ASRC component"
+	default y
+	help
+	  Select for Asynchronous sample rate conversion (ASRC) component
+
+if COMP_ASRC
+
+choice
+        prompt "ASRC down sampling conversions set"
+        default COMP_ASRC_DOWNSAMPLING_FULL
+
+config COMP_ASRC_DOWNSAMPLING_FULL
+	bool "Full downsampling conversions support"
+	select ASRC_SUPPORT_CONVERSION_24000_TO_08000
+	select ASRC_SUPPORT_CONVERSION_24000_TO_16000
+	select ASRC_SUPPORT_CONVERSION_48000_TO_08000
+	select ASRC_SUPPORT_CONVERSION_48000_TO_11025
+	select ASRC_SUPPORT_CONVERSION_48000_TO_12000
+	select ASRC_SUPPORT_CONVERSION_48000_TO_16000
+	select ASRC_SUPPORT_CONVERSION_48000_TO_22050
+	select ASRC_SUPPORT_CONVERSION_48000_TO_24000
+	select ASRC_SUPPORT_CONVERSION_48000_TO_32000
+	select ASRC_SUPPORT_CONVERSION_48000_TO_44100
+	help
+	  This option enables all supported downsampling conversions
+	  for the asynchronous sample rate conversion. All the upsampling
+	  conversions use the same filter coefficients set so there is no
+	  configuration options for them.
+
+config COMP_ASRC_DOWNSAMPLING_CUSTOM
+	bool "Custom downsampling conversions support"
+	help
+	  This option used to manually select each downsampling ratio.
```

Comment:
I like that verbosity in descriptions.

Reply:
OK, I'll read through and see if there's anything missing that's worth to add.